CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Securit
 
 ## Unreleased
 
+## [370.2] - beta
```

Comment:
I believe we usually do not specify the version here - we usually just put everything into "unreleased" and Kurt changes it to a version during the deploy. But as he approved I guess this is not an issue.

Reply:
Good to know, next time I'll just leave ## Unreleased